tests/modules/keyspace_events.c

```diff
@@ -0,0 +1,90 @@
+/* This module is used to test the server keyspace events API.
+ *
+ * -----------------------------------------------------------------------------
+ *
+ * Copyright (c) 2019, Salvatore Sanfilippo <antirez at gmail dot com>
```

Comment:
LOL, with copy paste Salvatore will live forever.
I think your name goes here (and note the data too)..